devlab/tests/config.py

```diff
@@ -1,3 +1,5 @@
+img_url = 'http://download.cirros-cloud.net/0.3.3/cirros-0.3.3-x86_64-disk.img'
```

Comment:
We use created images to spawn vms. For testing migration we need more than 1 image.